tests/integration/conftest.py

```diff
@@ -93,22 +110,77 @@ async def app(
     # mypy doesn't see that ActiveStatus has a name
     await model.wait_for_idle(status=ActiveStatus.name)  # type: ignore
 
+    # test that the application is online
+    for _ in range(10):
+        try:
+            response = requests.get(
+                f"http://{await get_mattermost_ip(ops_test, application)}:8065",
+                timeout=5
+            )
+            if response.status_code == 200:
+                break
+        except:
```

Comment:
This is quite dangerous, it would, for example, result in KeyboardInterrupt getting ignored as well. Should this be more specific? There is also a nice contextlib.suppress context manager: https://docs.python.org/3/library/contextlib.html#contextlib.suppress

Reply:
I specified the type of errors that I'm handling